tests/mmm/test_multidimensional.py

```diff
@@ -882,10 +883,14 @@ def test_mmm_with_events(
     )
     assert len(mmm.mu_effects) == 1
 
+    df_events_with_country = df_events.copy()
+    df_events_with_country["country"] = "A"
     mmm.add_events(
-        df_events,
+        df_events_with_country,
         prefix="another_event_type",
-        effect=create_event_effect(prefix="another_event_type"),
+        effect=create_event_effect(
+            prefix="another_event_type", dims=("country", "another_event_type")
+        ),
     )
```

Comment:
I wanted to enable specifying dimensions for the EventEffect, but kept the default behaviour unchanged.